docs/core/stories.rst

```diff
@@ -49,13 +52,73 @@ This is what we call a **story**.
   in the format ``intent{"entity1": "value", "entity2": "value"}``.
 - Actions executed by the bot are shown as lines starting with ``-``
   and contain the name of the action.
-- Events returned by an action are on lines immediately after that
-  action. For example, if an action returns a ``SetSlot`` event,
-  this is shown as the line ``- slot{"slot_name": "value"}``
+- Events returned by an action are on lines immediately after that action.
+  For example, if an action returns a ``SlotSet`` event, this is shown as
+  ``slot{"slot_name": "value"}``.
+
+Let's now take a slightly more detailed look at each of these components,
+along with some things you should keep in mind while using them to write
+your own stories.
+
+User Messages
+~~~~~~~~~~~~~
+While writing stories, you do not have to deal with the specific contents of
+the messages that the users send. Instead, you can take advantage of the output
+from the NLU pipeline, which lets you use just the combination of an intent and
+entities to refer to all the possible messages the users can send to mean the
+same thing.
+
+It is important to include the entities here as well because the policies learn
+to predict the next action based on a *combination* of both the intent and
+entities (you can, of course, customize this behavior using the
```

Comment:
```suggestion
entities (you can, however, change this behavior using the
```

maybe something like "the existence of entities" since the value doesn't influence action prediction, just whether or not the entity was picked up.

Reply:
Not if it's categorical, boolean, etc.

Reply:
No, that's for slots, not entities

Reply:
Yeah, but the slots are populated from the entities, right?